### PR TITLE
Minor test cleanup

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Minor test cleanup

### CODE REVIEW

File: tests/phpunit/CRM/Event/Form/Registration/ConfirmTest.php [hunk 1]
```diff
@@ -10,20 +10,9 @@ class CRM_Event_Form_Registration_ConfirmTest extends CiviUnitTestCase {
...
-
-  /**
-   * Should financials be checked after the test but before tear down.
-   *
-   * Ideally all tests (or at least all that call any financial api calls ) should do this but there
-   * are some test data issues and some real bugs currently blocking.
-   *
-   * @var bool
-   */
-  protected $isValidateFinancialsOnPostAssert = TRUE;
```
Comment (eileenmcnaughton) [inline tests/phpunit/CRM/Event/Form/Registration/ConfirmTest.php:22]: same as parent now

File: tests/phpunit/CRM/Event/Form/Registration/ConfirmTest.php [hunk 2]
```diff
@@ -788,7 +777,10 @@ public function testSubmitNonPrimaryEmail(): void {
   protected function submitForm(int $eventID, array $params): void {
     $_REQUEST['id'] = $eventID;
     $form = $this->getFormObject('CRM_Event_Form_Registration_Confirm', []);
-    $form->set('params', [$params]);
+    if (!isset($params[0])) {
```
Comment (eileenmcnaughton) [inline tests/phpunit/CRM/Event/Form/Registration/ConfirmTest.php:780]: More flexible

**Comment by demeritcowboy** [comment]
The PR title might no longer be correct.

```
not ok 714 - Error: CRM_Contact_Page_View_UserDashBoardTest::testEventDashboard
not ok 853 - Error: CRM_Contribute_Form_Task_InvoiceTest::testInvoiceForLineItems
not ok 1004 - Error: CRM_Core_BAO_ActionScheduleTest::testEventTypeRegistrationDate
not ok 1414 - Error: CRM_Core_CopyTest::testEventCopy
not ok 1820 - Failure: CRM_Event_Form_Registration_ConfirmTest::testWaitlistRegistrationContactIdParam
  ---
  message: 'Failure in api call for PaymentProcessor create:  DB Error: already exists'
  severity: fail
  ...
  not ok 1821 - Error: CRM_Event_Form_Registration_ConfirmTest::testTaxMultipleParticipant
not ok 1822 - Error: CRM_Event_Form_Registration_ConfirmTest::testOnlineRegNoPrice
not ok 1823 - Error: CRM_Event_Form_Registration_ConfirmTest::testAssignProfiles
not ok 1824 - Failure: CRM_Event_Form_Registration_ConfirmTest::testNoteSubmission
  ---
  message: 'Failure in api call for UFGroup create:  DB Error: already exists'
  severity: fail
  ...
  not ok 1825 - Error: CRM_Event_Form_Registration_ConfirmTest::testSubmitNonPrimaryEmail
not ok 1826 - Error: CRM_Event_Form_Registration_RegisterTest::testMinValueForPriceSet
```

**Comment by eileenmcnaughton** [comment]
ah yeah - I changed the return array in the end too  - will fix....

**Comment by eileenmcnaughton** [comment]
ah - permissions = FALSE

**Comment by demeritcowboy** [comment]
All the same test fails except the first one.

**Comment by eileenmcnaughton** [comment]
@demeritcowboy - looks like I got there

**Comment by eileenmcnaughton** [comment]
thanks @seamuslee001

**Pull request merged**